capstone/scripts/update_snippets.py

```diff
@@ -1,181 +0,0 @@
-import io
-import csv
-from collections import defaultdict
-
-from django.db import connections
-from django.db.models import Count, Q
-from capdb.models import Reporter, Jurisdiction, CaseMetadata, Snippet, Court
-import json
-from capweb.templatetags.api_url import api_url
-from tqdm import tqdm
-
-
-def update_all():
-    update_map_numbers()
-    cases_by_jurisdiction_tsv()
-    cases_by_reporter_tsv()
-    cases_by_decision_date_tsv()
-    search_reporter_list()
```

Comment:
These tsv snippets were no longer used, huh, in favor of static files last updated three years ago? 😬 I think we want to keep the code for these but have a task to write them to `/download/statistics/`, probably at the same time we do a bulk data export.

Reply:
> Thanks for working on this. The snippets part still needs some work to be performant here (see comments), including figuring out how to write statistics files to the downloads section, so could you maybe split the API fix out into a separate PR? Like a `git merge --no-commit --no-ff` of this branch onto the latest develop, and then send PRs for the two blocks of work separately.
> 
> You're right that doing the caching in the view would remove some moving parts. I think you would just do something like this:
> 
> ```
> # capweb.helpers:
> 
> @cache_func(lambda: "search_jurisdiction_list", settings.CACHED_JSON_OBJECT_TIMEOUT)
> def search_jurisdiction_list():
>     ...
> 
> # view:
>     ...
>     'search_jurisdiction_list': search_jurisdiction_list(),
> ```
> 
> Strikes me as a little cleaner. 🤷‍♂️
> 
> The big thing here is we can't do long-running queries like the one in `map_numbers` this way, and I don't think we want to delete the snippets for the statistics tsvs and be left serving the static ones generated in 2018. Let's fix both of those by figuring out a clean way to dump statistics in the /download/ section of the site ... come to think of it, maybe the clean way is to add a final step to `scripts.export.init_export` that creates a statistics folder as a subfolder of a bulk export? This might also be a good opportunity to rethink what wants to go in that folder as well.

Man— I scoured the code for current uses of these snippets and totally missed that. I haven't even thought about these things since I wrote the initial script in like 2017. Damn. I assumed it was rolled up into our other data export functionality. I'll restore this processing code and have a task write them out to files. Easy fix.